nixos/modules/programs/nano.nix

```diff
@@ -12,16 +12,16 @@ in
 
       nanorc = lib.mkOption {
         type = lib.types.lines;
-        default = "";
+        default = ''
+          set nowrap
+          set tabstospaces
+          set tabsize 2
+          include "${pkgs.nano}/share/nano/*.nanorc"
```

Comment:
@gnidorah I took that to mean that perhaps the include should be addded unconditionally